config/config.go

```diff
@@ -794,7 +801,7 @@ type AuthHandlerConfig struct {
 	Config rutils.AttributeMap `json:"config"`
 }
 
-// Validate ensures all parts of the config are valid.
+// Validate ensures all parts of the config are valid. If it exists, updates ExternalAuthConfig's ValidatedKeySet once validated.
```

Comment:
Similar remark to `ExternalAuthConfig`. The `config.Handlers` object is an unstructured attribute map. Any basic examples of its shape would be a great help here.

Reply:
updated with a basic example

Reply:
Thanks! This is way more complex than I thought it'd be. The example will be a great aid.